common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -2176,6 +2178,14 @@ object CelebornConf extends Logging {
       .timeConf(TimeUnit.MILLISECONDS)
       .createWithDefaultString("30s")
 
+  val RACK_RESOLVE_REFRESH_INTERVAL: ConfigEntry[Long] =
+    buildConf("celeborn.master.rack.resolve.refresh.interval")
```

Comment:
note: dot is used to split configuration namespace, not the word

Reply:
May be celeborn.master.rackResolve.refresh.interval?

Reply:
done